parse/kuneiform/grammar/KuneiformLexer.g4

```diff
@@ -100,19 +102,26 @@ fragment DIGIT: [0-9];
 // STMT_MODE captures actions and procedures.
 // it is used to allow us to correctly parse action / procedure bodies
 mode STMT_MODE;
+    COMMENT:   ( '//' ~[\r\n]* '\r'? '\n'
+        | '/*' .*? '*/'
```

Comment:
In `.*?`, what is the `?` for?  At least in regexp, `*` means zero or more.

Reply:
This seems to be the way of doing block comments in antlr. I noticed Gavin had used it before, chatgpt used it, others use it: https://stackoverflow.com/questions/12898052/antlr-how-to-skip-multiline-comments